boot_node/src/lib.rs

```diff
@@ -48,11 +48,8 @@ pub fn run(
         log::Level::Error => drain.filter_level(Level::Error),
     };
 
-    let logger = Logger::root(drain.fuse(), o!());
-    let _scope_guard = slog_scope::set_global_logger(logger);
-    slog_stdlog::init_with_level(debug_level).unwrap();
+    let log = Logger::root(drain.fuse(), o!());
```

Comment:
@AgeManning had to remove this to make the bootnode logging not panic. It seems to work alright, any idea why this is here?

Reply:
I think they needed to be there in order to get the logs out to display inside slog. 

If you run a boot node, do we still get logs to the terminal?

Reply:
yeah I get them alright. Sample:
```
_id: PeerId("16Uiu2HAm9TFzGRAgJKVyfZ54nDbCYNwEAFnpz6ybjM7KaVPLJZpy"), ipv6_address: None, ipv4_address: Some(3.64.117.223:9100)
Dec 12 23:23:45.909 INFO Executing bootstrap query...
2023-12-12T23:23:45.909324Z  INFO discv5::service: Discv5 Service started
2023-12-12T23:23:45.909386Z  INFO discv5::service: Ip4
Dec 12 23:23:50.126 INFO Server metrics 
```